test/services/promo_registrar_test.rb

```diff
@@ -63,4 +57,7 @@ class PromoRegistrarTest < ActiveJob::TestCase
       PromoRegistrar.new(publisher: publisher).perform
     end
   end
+
+  test "" do
```

Comment:
Deadcode?

Reply:
Hm, I should probably write some more tests.